packages/apollo-gateway/src/index.ts

```diff
@@ -161,6 +161,28 @@ type WarnedStates = {
   remoteWithLocalConfig?: boolean;
 };
 
+export const GCS_RETRY_COUNT = 5;
+
+export function getDefaultGcsFetcher() {
+  return fetcher.defaults({
+    cacheManager: new HttpRequestCache(),
+    // All headers should be lower-cased here, as `make-fetch-happen`
+    // treats differently cased headers as unique (unlike the `Headers` object).
+    // @see: https://git.io/JvRUa
+    headers: {
+      'user-agent': `apollo-gateway/${require('../package.json').version}`,
+    },
+    retry: {
+      retries: GCS_RETRY_COUNT,
+      // 1 second
+      minTimeout: 1000,
+      // 60 seconds - but this shouldn't be reachable based on current settings
+      maxTimeout: 60 * 1000,
```

Comment:
I think I'll ultimately suggest removing this unless you think it conveys some intent (e.g., to never breach a specific time for other internal implementation detail reasons that live elsewhere, even in the event that someone comes across this code in the future and tweaks the `retries` or timeout values).  Even in that case though, I think it should be even more explicitly commented.  Right now this is a mathematical impossibility, so it's perhaps a bit confusing.

Alternatively, we could at least include `factor` with its default value for clarity as to _why_ it's impossible? That would also help explain the expectation here. ([The default is `2`](https://www.npmjs.com/package/retry#retrytimeoutsoptions) so this calculates out to attempts at 0 (immediate), 1, 3, 7, 15, and 31 seconds elapsed).  In fact, perhaps we should just put that in a comment so it's immediately clear?

Reply:
Agree on all counts! Resolved in 5f11639